core/dbt/config/runtime.py

```diff
@@ -107,9 +125,9 @@ def get_profile(
         return load_profile(
             project_root,
             cli_vars,
-            args.profile,
-            args.target,
-            args.threads,
+            getattr(args, "profile", None),
+            getattr(args, "target", None),
+            getattr(args, "threads", None),
```

Comment:
Safe access to these attributes was [knowingly removed](https://github.com/dbt-labs/dbt-core/compare/feature/click-cli#diff-5231952f5e1234a0ee90a5f124a45ca1ebb429a9d606c76d01b2a32707d068ccL422-L425) in the feature/click-cli branch and should work with the new framework. Restoring safe access here and elsewhere on this branch to get backwards compatibility with the old framework for the purposes of testing.